doc/source/whats_new.rst

```diff
@@ -137,13 +139,13 @@ API
    - find_events and read_events functions have a new parameter `mask` to set some bits to a don't care state by `Teon Brooks`_
 
    - New channels module including layouts, electrode montages, and neighbor definitions of sensors which deprecates
-    ``mne.layouts`` by `Denis Engemann`_
+    `mne.layouts` by `Denis Engemann`_
```

Comment:
check the doc `Makefile`, there is an option that allows you to skip building the pygraphviz visual. IIRC it also skips running examples, so if you want to run examples but not build the visual, just make a new entry in the `Makefile` (the formula can be inferred from the other entries).

What OS are you on, anyway? In Ubuntu it's just `sudo apt-get install graphviz` and then pygraphviz installs easily, maybe also by `apt-get` but I don't remember for sure.